traffic_portal/test/integration/PageObjects/CacheGroup.po.ts

```diff
@@ -117,16 +116,15 @@ export class CacheGroupPage extends BasePage {
         if (cachegroup.Type == "EDGE_LOC") {
             const name = cachegroup.FailoverCG + this.randomize;
             await this.txtFailoverCG.click();
-            if(await browser.isElementPresent(element(by.xpath(`//select[@name="fallbackOptions"]//option[@label="`+ name + `"]`)))){
-                await element(by.xpath(`//select[@name="fallbackOptions"]//option[@label="`+ name + `"]`)).click();
-            }else{
+            if (await browser.isElementPresent(element(by.xpath(`//select[@name="fallbackOptions"]//option[@label="` + name + `"]`)))) {
```

Comment:
Also, I wouldn't ask you to change it in this PR, but I think most people are more familiar with CSS selectors than XPath, so this could be written as:
```typescript
by.css(`select[name="fallbackOptions"] > option[label="${name}"]`)
```

Reply:
Fixed.